modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,257 @@
+
+:FeatureName: Creating a ``NodeNetworkConfigurationPolicy` object that includes a customized `br-ex` bridge
+include::snippets/technology-preview.adoc[]
+
+This feature supports the following tasks:
+
+* Modifying the maximum transmission unit (MTU) for your cluster.
+* Rollback your Kubernetes OVN plugin configuration to an earlier state.
+* Switch to a different bond interface, such as MIImon (Media Independent Interface Monitor), bonding mode, or Quality of Service (QoS).
+* Updating DNS values.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+Consider the following use cases for creating manifest object that includes a customized `br-ex` bridge:
+
+* You want to make postinstallation changes to the bridge, such as changing the Open vSwitch (OVS) or OVN-Kubernetes `br-ex` bridge network. The `configure-ovs.sh` shell script does not support making postinstallation changes to the bridge.
+* You want to deploy the bridge on a different interface than the interface available on a host or server IP address.
+* You want to make advanced configurations to the bridge that are not possible with the `configure-ovs.sh` shell script. Using the script for these configurations might result in the bridge failing to connect multiple network interfaces and facilitating data forwarding between the interfaces.
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+[NOTE]
+====
+If you require an environment with a single network interface controller (NIC) and default network settings, use the `configure-ovs.sh` shell script.
+====
+
+After you install {op-system-first} and the system reboots, the Machine Config Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration. The Operator prevents `br-ex` bridge configuration conflicts by sending a signal to the `configure-ovs.sh` shell script to not configure the `br-ex` bridge.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Prerequisites
+
+
+.Procedure
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a `MachineConfig` manifest file and define a customized `br-ex` bridge network configuration similiar to the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+  name: 10-br-ex-worker <2>
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents: <3>
+          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_config>
+          mode: 0644
+          overwrite: true
+          path: /etc/nmstate/openshift/<hostname>.yml
+---
+spec:
```

Comment:
We only need to keep the interfaces field and everything below it. The rest of this is NNCP syntax that isn't used here.

Reply:
:heavy_check_mark: